2.py

```diff
@@ -0,0 +1,23 @@
+import random
+
+
+n = int(input("Enter a number (0 < n < 30): "))
+
+
+if 0 < n < 30:
+
+    numbers = [random.randint(0, 1000) for _ in range(n)]
```

Comment:
ასეც იმუშავებს უბრალოდ ლისტის გამოყენების გარეშეც შეგეძლო დაგეწერა, პირდაპირ for ციკლში დაგეგენერირებინა შემთხვევითი რიცხვი და შეგემეწმებიდა მაქსიმალური ხო არაა